src/pages/GetStarted/GetStartedCauses.vue

```diff
@@ -184,18 +182,49 @@ export default {
 <style lang="scss" scoped>
 @import 'settings';
 
-.page-content {
-	padding: 1.625rem 0;
-}
-
 .get-started {
+	&__form {
+		padding: 0.85rem;
+		margin: 2rem auto;
+		background-color: $white;
+		border-radius: 1rem;
+		width: 98%;
```

Comment:
what does this do?

Reply:
For mediumish screens, keeps it to being full width so the darker gray background still shows through on all sides